include/ensmallen_bits/nsga2/nsga2.hpp

```diff
@@ -0,0 +1,323 @@
+/**
+ * @file nsga2.hpp
+ * @author Sayan Goswami
+ *
+ * NSGA-II is a multi-objective optimization algorithm, widely used in
+ * many real-world applications. NSGA-II generates offsprings using
+ * crossover and mutation and then selects the next generation according
+ * to nondominated-sorting and crowding distance comparison.
```

Comment:
```suggestion
 * to non-dominated-sorting and crowding distance comparison.
```

I think you can also go with nondominatedsorting, but I think this is easier to read?

Reply:
I'd agree, `non-dominated-sorting` seems easier to understand to me.

Reply:
Fixed.